pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/EntryFilters.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.common.policies.data;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class EntryFilters {
+    /**
+     * The description of the entry filter to be used for user help.
+     */
+    private String description;
+
+    /**
+     * The class name for the entry filter.
+     */
+    private String entryFilterNames;
+
+    /**
+     * The directory for all the entry filter implementations.
+     */
+    private String entryFiltersDirectory;
```

Comment:
This is a parameter that is only useful only in ServiceConfiguration (broker.conf) because it refers to the local filesystem of every specific broker.

I think we should drop this from here